### PR TITLE
Test revisions

### PR DESCRIPTION
**Pull request by subelsky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ This project is a simple Rails app intended to help you demonstrate:
 * Problem-solving abilities
 * Ability to use the [Git version control system](https://git-scm.com/)
 
-Your goal is to add functionality and fix bugs so that the app's [RSpec test suite](spec/) passes.
+## Scenario
+
+Your goal is to add functionality and fix bugs so that the app's [RSpec test suite](spec/) passes. This will involve modifying both application code (where functionality is missing or broken) and RSpec code (where the functionality is correct, but the test is not written correctly).
```
Comment (hoenth) [inline README.md:16]: Didn't we change this to be that all the specs are correct, so they won't have to guess whether it is a spec issue or a code issue?

**Review by hoenth — COMMENTED** [review]
These look great. I'd like to get Pavan to complete them, to see what kind of baseline we get from him.

File: README.md (hunk 1 above)
Comment (subelsky) [reply README.md:16]: I remember discussing that but didn't remember what we decided. I'll make that fix now

File: app/models/question.rb [hunk 2]
```diff
@@ -11,10 +11,10 @@ def options_string=(option_string)
       if (found_option = options.select { |o| o.name.downcase == option.downcase }.first).present?
         # if there already existed an option with the same name (case insenstive)
         # update the position and name (case sensitive)
-        found_option.update(position: index + 1, name: option)
+        found_option.update(name: option)
```
Comment (subelsky) [inline app/models/question.rb:14]: ^^^ @hoenth this 'breaks' the spec that I 'fixed' below

**Pull request merged**